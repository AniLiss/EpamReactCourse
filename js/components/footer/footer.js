import React from 'react';
import {Logo} from '../logo/logo';

export const Footer = () => {
    return (
        <footer className="footer">
            <Logo />
        </footer>
    )
};
