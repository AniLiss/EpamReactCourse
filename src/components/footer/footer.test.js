import {mount} from 'enzyme';
import React from 'react';
import {Footer} from './footer';

describe('Footer', () => {
    it('is expected to render footer correct', () => {
        const wrapper = mount(<Footer />);
        expect(wrapper).toMatchSnapshot();
    })
});
