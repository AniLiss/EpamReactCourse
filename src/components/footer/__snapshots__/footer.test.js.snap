// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer is expected to render footer correct 1`] = `
<Footer>
  <footer
    className="footer"
  >
    <Logo>
      <h1
        className="logo"
      >
        netflixroulette
      </h1>
    </Logo>
  </footer>
</Footer>
`;
